<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>algorithmDlg</class>
 <widget class="QWidget" name="algorithmDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Calibration:  Algorithm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>142</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Algo.  Model&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>229</y>
     <width>133</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;DF Control&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>316</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Moving Avg.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Model_lineEdt">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>146</y>
     <width>159</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="DFEnable_lineEdit">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>233</y>
     <width>159</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="MovAvg_lineEdit">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>320</y>
     <width>159</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="calculationmodelTip">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>142</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digitalfilteringTip">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>229</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slidingaveragetimeTip">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>316</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>402</y>
     <width>130</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>402</y>
     <width>130</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pageDown">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>402</y>
     <width>130</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>402</y>
     <width>130</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateTime">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Model_change">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>142</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DFEnable_change">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>229</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="movingaverage_change">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>316</y>
     <width>85</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
